File: testprojects/scala_main/pipelines/DONOT_OPEN_SCALA/code/src/main/scala/org/main/scla_dep_mgmt/graph/Repartition_1.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Repartition_1 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.repartitionByRange(10.toInt,
                          col("c_timestamp").asc,
                          col("`c   short  --`").asc,
                          col("`c  date`").desc,
                          concat(col("c_timestamp"), col("`c  date`")).desc
    )

}

File: testprojects/python_main/pipelines/PERF_UNITEST_GENERATE/code/perf_unitest_generate/graph/Repartition_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from perf_unitest_generate.config.ConfigStore import *
from perf_unitest_generate.udfs.UDFs import *

def Repartition_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.coalesce(10)
